docker-compose.yaml

```diff
@@ -30,6 +30,7 @@ services:
       HBASE_CLIENT_SCANNER_TIMEOUT_PERIOD_MS: "60000"
       HBASE_PAUSE_MILLISECONDS: "50"
       HBASE_RETRIES": "3"
+      HBASE_REPLICA_ID: "2"
```

Comment:
does this need copying over to the infra too?

Reply:
It will do ye. I haven't even linked this in the PR channel yet ha!

Reply:
It has a default though so can be released without it